src/test/java/de/retest/recheck/ui/diff/DurationDifferenceTest.java

```diff
@@ -0,0 +1,46 @@
+package de.retest.recheck.ui.diff;
+
+import static de.retest.recheck.XmlTransformerUtil.toXmlFragmentViaJAXB;
+import static de.retest.recheck.util.ApprovalsUtil.verifyXml;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class DurationDifferenceTest {
+
+	@Test
+	public void same_duration() {
+		final DurationDifference difference = DurationDifference.differenceFor( 70, 70 );
+		assertThat( difference ).isNull();
+	}
+
+	@Test
+	public void expected_is_longer_but_below_threshold() {
```

Comment:
If you rely on the threshold, you might want to set it explicitly as system property here.